programs/voter-stake-registry/src/state/registrar.rs

```diff
@@ -26,6 +26,7 @@ const_assert!(std::mem::size_of::<Registrar>() == 5 * 32 + 4 * 120 + 8 + 1 + 31)
 
 impl Registrar {
     pub fn clock_unix_timestamp(&self) -> i64 {
+        //#[soteria(ignore)]
```

Comment:
Excellent that this works now!

For this specific issue, I think we should just checked_add().unwrap(), since `time_offset` is guaranteed to be 0 in all non-test deployments.